### PR TITLE
fix:Google oauth

### PR DESCRIPTION
**Pull request by Pranay105**
- Updated the content as per the feedback

### CODE REVIEW

File: website/docs/getting-started/setup/instance-configuration/authentication/google-login.md [hunk 1]
```diff
@@ -37,21 +37,10 @@ To enable Google Sign-in, go to the [Google API console](https://console.cloud.g
 ## Configure Appsmith instance  
 
 :::caution
-* The Client ID and Client Secret entered aren't verified by any means. Please test it out manually before disabling the Form login authentication method. If Form login is disabled before testing the Google authentication configuration setup, it may lead to the user getting stuck on the login screen if the Google configuration is invalid. You’ll have to update the environment variables manually to fix this, as you can't edit it via the Admin settings page since you won’t be logged in anymore.
-
+* Please manually test the Client ID and Client secret before disabling the Form login authentication. If the google configuration is invalid, the user is stuck on the login screen. To fix this, you’ll have to update the environment variables manually.
```
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/authentication/google-login.md:40]: Can we simplify this note? 
```suggestion
* Test the Client ID and Client Secret before disabling Form login on Appsmith. This ensures that users can access Appsmith with their Google credentials. To fix an invalid Google configuration, update the environment variables.
```

File: website/docs/getting-started/setup/instance-configuration/authentication/google-login.md [hunk 2]
```diff
@@ -37,21 +37,10 @@ To enable Google Sign-in, go to the [Google API console](https://console.cloud.g
...
 * If you are running appsmith behind an ELB / Proxy, [configure Nginx](/help-and-support/troubleshooting-guide/deployment-errors#oauth-sign-up-not-working).
```
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/authentication/google-login.md:41]: These two pieces of information do not tie up together. Maybe a rewrite could be:
```suggestion
* To fix signup issues, if you are running appsmith behind an ELB / Proxy, see [OAuth Sign up not working](/help-and-support/troubleshooting-guide/deployment-errors#oauth-sign-up-not-working).
```
P.S. - can we get this confirmed by the technical team?

File: website/docs/getting-started/setup/instance-configuration/authentication/google-login.md [hunk 3]
```diff
@@ -37,21 +37,10 @@ To enable Google Sign-in, go to the [Google API console](https://console.cloud.g
...
-There are two ways to configure the self-hosted Appsmith instance - Admin Settings and Environment Variables:
-
-**On Admin Settings screen (recommended):**  On your self-hosted instance, navigate to the [Admin Settings page](/getting-started/setup/instance-configuration/). 
-
-* Under the **Authentication** tab,  provide the Client ID and Client Secret generated in the previous steps.
-* Once you've entered your Client ID and Client Secret, click the **Save & Restart** button at the bottom of the page. This prompts your Appsmith instance with the new configuration in effect.
-
-![](/img/as_google_auth_config.png)
-
-**Using environment variables:** Update the values for the following keys in your configuration file.
```
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/authentication/google-login.md:54]: Let's add a short intro for Configuring Appsmith Instance, which tells users that there are below ways in which the instance could be configured:
* Admin Settings(Add links to its subsection)
* Environment Variables(Add links to its subsection)

File: website/docs/getting-started/setup/instance-configuration/authentication/google-login.md [hunk 4]
```diff
@@ -37,21 +37,10 @@ To enable Google Sign-in, go to the [Google API console](https://console.cloud.g
...
+On the machine hosting the Appsmith instance, find your `docker.env` file located in `<project-root>/stacks/configuration/`. Update the values for the following keys:
```
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/authentication/google-login.md:43]:
```suggestion
Navigate to the `docker.env` file located in `<PROJECT_ROOT>/stacks/configuration/`. Update the values for the following keys:
```

File: website/docs/getting-started/setup/instance-configuration/authentication/google-login.md [hunk 5]
```diff
@@ -66,4 +55,9 @@ There are two ways to configure the self-hosted Appsmith instance - Admin Settin
   # ******************************
    ```
```
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/authentication/google-login.md:57]: Create a subhead for Admin Settings.

File: website/docs/getting-started/setup/instance-configuration/authentication/google-login.md (hunk 4 above)
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/authentication/google-login.md:43]: Create a subhead for Environment Variables.

**Review by jnikhila — CHANGES_REQUESTED** [review]

**Comment by infinitetrooper** [comment]
@jnikhila @Pranay105  - We're adding the Javascript Origin URI and Redirect URI in the Google OAuth config screen on Appsmith so the copy-paste experience is smoother (Image attached below), this will go in the next release so, can we wait for a little before we merge this and include the updated screenshot and process also?

Screenshot 2023-03-10 at 5 22 31 PM

**Comment by jnikhila** [comment]
> @jnikhila @Pranay105 - We're adding the Javascript Origin URI and Redirect URI in the Google OAuth config screen on Appsmith so the copy-paste experience is smoother (Image attached below - there's a typo there), this will go in the next release so, can we wait for a little before we merge this and include the updated screenshot and process also?
> 
> [appsmithorg/appsmith#21170](https://github.com/appsmithorg/appsmith/issues/21170)
> 
> Screenshot 2023-03-10 at 5 22 31 PM

@infinitetrooper - let's add a new ticket for this change, and we can merge the changes along with #[21170](https://github.com/appsmithorg/appsmith/issues/21170)

**Pull request merged**